packages/web-app-files/src/components/Search/List.vue

```diff
@@ -16,11 +16,14 @@
       :target-route="resourceTargetLocation"
       :are-paths-displayed="true"
       :are-thumbnails-displayed="displayThumbnails"
-      :has-actions="false"
+      :has-actions="true"
       :is-selectable="false"
       @fileClick="$_fileActions_triggerDefaultAction"
       @rowMounted="rowMounted"
     >
+      <template #contextMenu="{ resource }">
+        <context-actions :items="[resource]" />
```

Comment:
This creates the context-actions for each row in the table. Could you check if you can tmp store the right clicked resource and use a `v-if` on `context-actions` to only create the context-actions in the DOM that were right-clicked last? Otherwise you create the context-actions for each and every file row in the DOM (even if they're not shown).